.github/workflows/update-docs-in-website.yml

```diff
@@ -0,0 +1,120 @@
+name: Update latest Bindings documentation in the website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "**/*.md"
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest Bindings documentation
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v3
+        with:
+          path: bindings
+      - name: Checkout Another repository
+        uses: actions/checkout@v3
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-bindings-docs-${{ github.sha }}
+      - name: Update edit-page-config.json
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const { writeFile } = require('fs').promises;
+            const configPath = './website/config/edit-page-config.json';
+            const checkSlug = 'reference/bindings/';
+            const slug = {
+              "value": checkSlug,
+              "href": "https://github.com/asyncapi/bindings/tree/master"
```

Comment:
it is not good as edit link is `https://github.com/asyncapi/bindings/tree/master/solace.mdx` and should be `https://github.com/asyncapi/bindings/blob/master/solace/README.md`

Reply:
We can't customise this specifically for some docs in website. We should follow a standardised way to render docs in other repos as well.

Reply:
ok, makes sense